### PR TITLE
Version 02

### PR DESCRIPTION
**Pull request by FightForDobro**
Добрый вечер, сделал домашние задания на вторник и четверг

### CODE REVIEW

**Comment by FightForDobro** [comment]
Added .gitignore file

File: Ushakov_Denys/01/Home_Work_1.py [hunk 1]
```diff
@@ -0,0 +1,1247 @@
...
+    main_menu_button = input('Enter here ──> ')
+
+    if not main_menu_button.isdigit() or int(main_menu_button) > 4:
+
+        print('Please enter number from 1-4')
+        time.sleep(1)
+
+        continue
+
+    if main_menu_button.isdigit():
```
Comment (medviediev) [inline Ushakov_Denys/01/Home_Work_1.py]: Should be else of the previous if.

File: Ushakov_Denys/01/Home_Work_1.py [hunk 2]
```diff
@@ -0,0 +1,1247 @@
+# Simple Black Jack v - Beta 1.0
+from random import randint as ri
```
Comment (medviediev) [inline Ushakov_Denys/01/Home_Work_1.py]: Bad naming "ri"

File: Ushakov_Denys/01/Home_Work_1.py [hunk 3]
```diff
@@ -0,0 +1,1247 @@
...
+                    print(f'You enter: {game_menu_button}')
+                    print('Enter 1 or 2')
+                    continue
+
+                else:
+                    break
+
+            # Game logic
+
+            if game_menu_button.lower() == '1':
```
Comment (medviediev) [inline Ushakov_Denys/01/Home_Work_1.py]: No need to use lower() with digits

File: Ushakov_Denys/02/Task_1.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+x = [i for i in new_string if i == 'x']
+
+# True condition
+if len(o) == len(x):
+
+    print(new_string)
+    print('True')
+
+# False condition
+elif o != x:
```
Comment (medviediev) [inline Ushakov_Denys/02/Task_1.py]: Should be just else

File: Ushakov_Denys/02/Task_3.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# List
+new_list = [1, 2, 3, 1, 2, 3, 1, 3, 3]
+
+# Calculate elements
+one = [i for i in new_list if i == 1]
+two = [i for i in new_list if i == 2]
+three = [i for i in new_list if i == 3]
```
Comment (medviediev) [inline Ushakov_Denys/02/Task_3.py]: Should be done for any number, not only 1, 2 or 3

**Pull request merged**